Builds/VisualStudio2015/RippleD.vcxproj

```diff
@@ -197,7 +197,7 @@
     <ClCompile>
       <PreprocessorDefinitions>HAVE_USLEEP=1;SOCI_CXX_C11=1;_WIN32_WINNT=0x6000;BOOST_NO_AUTO_PTR;DEPRECATED_IN_MAC_OS_X_VERSION_10_7_AND_LATER;NDEBUG;NOMINMAX;OPENSSL_NO_SSL2;WIN32_CONSOLE;_CRT_SECURE_NO_WARNINGS;_SCL_SECURE_NO_WARNINGS;_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <AdditionalIncludeDirectories>..\..\build\proto;..\..\src;..\..\src\beast;..\..\src\beast\extras;..\..\src\beast\include;..\..\src\protobuf\src;..\..\src\protobuf\vsprojects;..\..\src\soci\include;..\..\src\soci\src;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <DisableSpecificWarnings>4800;4244;4267;4018</DisableSpecificWarnings>
+      <DisableSpecificWarnings>4800;4244;4267;4503;4018</DisableSpecificWarnings>

```

Comment:
so, I would assume that this vcxproj would be removed if/when we moved to CMake, yes?  My recollection is that there is no way to explicitly set some of these very specific VS settings like "Disable Specific Warnings" in the CMake generated project, but warnings in particular can be handled with that pragma idiom:

    #pragma warning( push )
    #pragma warning( disable : 4800 )
    #pragma warning( disable : nnnn )
    ...
    #pragma warning( pop )